packages/synapse-react-client/src/components/dataaccess/AccessRequirementDashboard.tsx

```diff
@@ -151,7 +140,6 @@ export function AccessRequirementDashboard(
             type="text"
             fullWidth
             placeholder="Search for an Access Requirement Name or ID"
-            value={nameOrID}
```

Comment:
Make text box components uncontrolled (so the user's text input is NOT debounced, even if the value stored in state is)